MetaMask/metamask-extension
# feat: re-add account Snaps toggle to experimental tab

## **Description**

This PR reverts some of the changes from #21652 and re-enables the **Add "account Snap"** toggle in the experimental settings.

## **Related issues**

Partialy reverts: #21652
Fixes: https://github.com/MetaMask/accounts-planning/issues/179

## **Manual testing steps**

1. Install MetaMask stable and perform the onboarding
2. Go to **...** > **Settings** > **Experimental** > **Snaps** > **Account Snaps**
3. The **Enable "Add account Snap (Beta)"** toggle should be visible
4. Enable the **Enable "Add account Snap (Beta)"** toggle
5. Go to **Account picker** > **Add account or hardware wallet**
6. The **Add account Snap** button should be visible

## **Screenshots/Recordings**



### **Before**

<img width="1042" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/68558152/5dba7cc2-016f-49b0-a045-07b71550971f">

### **After**

https://github.com/MetaMask/metamask-extension/assets/68558152/10cd5187-bc7c-42b8-9b2b-a95481645ef1

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
